src/itype.h

```diff
@@ -441,7 +441,7 @@ struct islot_gunmod : common_ranged_data {
     int sight_dispersion = -1;
 
     /**
-     *  If set (non-zero) mod functions as sight when recoil above mod @ref sight_dispersion */
+     *  If set (non-negative) mod functions as sight when recoil above mod @ref sight_dispersion */
```

Comment:
This is wrong - you default the value to 10, meaning it is always set.
Either comment what it actually does and comment that `sight_dispersion` determines what is and what isn't a sight or change the default value to -1.

Either way, fixing that `sight_dispersion` lack of comment above would also help.